src/kbmod/search/image_kernels.cu

```diff
@@ -236,9 +236,88 @@ extern "C" void deviceGrowMask(int width, int height, float *source,
     checkCudaErrors(cudaFree(deviceResultImg));
 }
 
+extern "C" __device__ __host__ pixelPos findPeakImageVect(int width, int height, float *img,
```

Comment:
I like how these inner functions are cross-compiled to cpu for testing! That's a great idea.